ui/temboardui/static/src/views/Configuration.vue

```diff
@@ -19,58 +22,134 @@ const resetParamName = ref("");
 const resetParamValue = ref("");
 
 onMounted(() => {
-  if (props.error_code != 0) {
-    showError(props.error_message);
-  }
+  getCategories();
 });
 
-function generatePopoverContent(row) {
-  let content = `<table><tr><td>Type:</td><td><b>${row["vartype"]}</b></td></tr>`;
-  if (row["unit"]) {
-    content += `<tr><td>Unit:</td><td><b>${row["unit"]}</b></td></tr>`;
+function getCategories() {
+  $.ajax({
+    url: "/proxy/" + props.address + "/" + props.port + "/pgconf/configuration/categories",
+    type: "GET",
+    async: true,
+    contentType: "application/json",
+    dataType: "json",
+    success: function (data) {
+      configurationCategories.value = data;
+      getConfiguration();
+    },
+    error: function (xhr) {
+      showError(xhr);
+    },
+  });
+}
+
+function getConfiguration(closeError = true) {
+  if (closeError) {
+    clearError();
   }
-  if (["integer", "real"].includes(row["vartype"])) {
-    content +=
-      `<tr><td>Minimum:</td><td><b>${row["min_val"]}</b></td></tr>` +
-      `<tr><td>Maximum:</td><td><b>${row["max_val"]}</b></td></tr>`;
+  let url = "";
```

Comment:
For me, it would be easier to read with:

``` javascript
let url = `/proxy/${props.address}/${props.port}/pgconf/configuration`
...
if (queryFilter.value == "") {
  url += `/category/${quotePlus(currentCat.value)}`;
}
```